packages/api/src/hooks/p2p/useAdvertList.ts

```diff
@@ -50,10 +50,11 @@ const useAdvertList = (
                 /** Indicates if the advertiser is currently online. */
                 is_online: Boolean(advert?.advertiser_details?.is_online),
                 /** Indicates that the advertiser was recommended in the most recent review by the current user. */
-                is_recommended: Boolean(advert?.advertiser_details?.is_recommended),
+                is_recommended:
+                    advert?.advertiser_details?.is_recommended === null
+                        ? null
+                        : Boolean(advert?.advertiser_details?.is_recommended),
             },
```

Comment:
maybe can add `has_not_been_recommended` attribute?
```suggestion
            has_not_been_recommended: advert?.advertiser_details?.is_recommended === null
            },
```

Reply:
I'll add it on my hook `useOrderReview` as well

Reply:
We should keep `is_` `has_` `should_` as boolean values, its also more clear for the developers on how this variable is represented:
```
if (data.has_not_been_recommended) {...}
```
is more clear than
```
if (data.is_recommended === null) {...}
```